collector/src/descriptor/kind/bridge_server_descriptor.rs

```diff
@@ -61,6 +119,29 @@ impl BridgeServerDescriptor {
             v.join("")
         };
 
+        let uptime = {
+            let v = take_uniq(&mut desc, "uptime", 1)?;
+            v[0].parse()?
+        };
+
+        let bandwidth = {
+            let v = take_uniq(&mut desc, "bandwidth", 3)?;
+            (
+                v[0].parse().unwrap(),
```

Comment:
please refrain from using unwraping when the result isn't provably Ok/Some, propagate an error instead